### DISCUSSION TITLE
greenlet version conflict

### DISCUSSION THREAD

**Discussion by sorokinal** [Kaliiiiiiiiii-Vinyzu/patchright#23] [Q&A]
Hello!
I'm encountering a version conflict error when trying to install Playwright 1.4. It seems that the undetected-playwright-patch (1.40.0.post1700587210000) depends on greenlet (3.0.1), while playwright (1.47.0) depends on greenlet (3.0.3), which causes an incompatibility between the two packages.

Would it be possible for you to update the version or provide guidance on how to resolve this issue?

**Comment by ttlns** [comment]
@sorokinal is this still the case//an issue?
